src/locales/en/translation.json

```diff
@@ -8,6 +8,7 @@
   "proposals_zero": "No proposals",
   "proposals_one": "{{count}} proposal",
   "proposals_other": "{{count}} proposals",
+  "proposals_all": "All proposals",

```

Comment:
I think we should call this `allProposals`. 
the `_one`, `_other` are for special use with `count` prop. (also `_zero` for `count:0`). But if you will use this as a normal key (like you are doing in /GuildSidebar.tsx `{t('proposals_all')}`) it should be `allProposals`.
Seee [i18next plurals doc](https://www.i18next.com/translation-function/plurals)

Reply:
Totally, makes more sense